Neos.Flow/Classes/ObjectManagement/ObjectManagerInterface.php

```diff
@@ -56,6 +57,7 @@ public function get($objectName, ...$constructorArguments);
      * @param string $objectName
      * @return bool
      * @see isRegistered
+     * don throw
```

Comment:
?
```suggestion
```

Reply:
hmm it was late and this was meant to be a todo - upsi.

The PsrContainer actual describes what kind of exceptions can be thrown, but we dont keep this api promise by throwing custom exceptions. Question is, if we should throw the right exceptions?

Reply:
I would suggest to address this in a separate PR so that this one stays a purely "cosmetic" change

Reply:
🤷‍♂️